### PR TITLE
Update dependencies to k8s-1.18.3

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@fedepaol FYI

**Comment by fedepaol** [comment]
Thanks @cynepco3hahue !

File: pkg/controller/performanceprofile/components/machineconfig/machineconfig.go [hunk 1]
```diff
@@ -83,7 +84,11 @@ func New(assetsDir string, profile *performancev1alpha1.PerformanceProfile) (*ma
 		return nil, err
 	}
 
-	mc.Spec.Config = *ignitionConfig
+	rawIgnition, err := json.Marshal(ignitionConfig)
```
Comment (fedepaol) [inline pkg/controller/performanceprofile/components/machineconfig/machineconfig.go:87]: Why this change?

Comment (cynepco3hahue) [reply pkg/controller/performanceprofile/components/machineconfig/machineconfig.go:87]: Because of the change under the machine config repository, I updated our deps to use release-4.5 instead of 4.4
